Tensile/TensileCreateLibrary.py

```diff
@@ -47,6 +47,16 @@
 import time
 from copy import deepcopy
 
+ # Translate GPU targets to filter filenames in Tensile_LOGIC directory
+mapArchitecture = {'all':'_','gfx000':'none', 'gfx803':'r9nano',
+  'gfx900':'vega10', 'gfx906':'vega20', 'gfx908':'arcturus'}
+
+
+def getArchitectureName(gfxName):
+  if gfxName in mapArchitecture:
+    return mapArchitecture[gfxName]
+  else:
+    return None
 
```

Comment:
This function is not used in this file. May be a better place to put this. Common?

Reply:
fixed.